.github/workflows/check-pr.yaml

```
@@ -95,4 +115,27 @@ jobs:
           if-no-files-found: ignore
           name: test-logs
           path: |
-            node-it/target/logs/*/
\ No newline at end of file
+            node-it/target/logs/*/
+
+  publish-snapshots:
+    name: Publish snapshot packages
+    needs: check-pr
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'publish-snapshots')
+    env:
+      PR_NUMBER: ${{ github.event.number }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+          cache: 'sbt'
+          gpg-private-key: ${{ secrets.OSSRH_GPG_KEY }}
+          gpg-passphrase: ${{ secrets.OSSRH_GPG_PASSPHRASE }}
+      - uses: sbt/setup-sbt@v1
+      - run: sbt -Dproject.version=$(awk -F '[".]' '{print $3"."$4"."$5+1}' version.sbt)-$PR_NUMBER-SNAPSHOT --batch ';publish'
```

```suggestion
      - run: sbt -Dproject.version=$(awk -F '[".]' '{print $3"."$4"."$5+1}' version.sbt)-$PR_NUMBER-SNAPSHOT --batch 'publish'
```
The heading ";" is not required in ";even;there;are;multiple;commands"